.gitignore

```
@@ -10,3 +10,5 @@ book/website/Pipfile\.lock
 
 workshops/boost-research-reproducibility-binder/workshop-presentations/*pptx
 .Rproj.user
+
+*.pyc
```

```suggestion
**/__pycache__/
```
Git-ignore the pycache folder rather than individual compiled files